CIP-Registration/CIP-Registration.md

```diff
@@ -0,0 +1,110 @@
+---
+CIP: TODO
+Title: Catalyst Registration Transaction Metadata Format
+Authors: Sebastien Guillemot <sebastien@emurgo.io>, Rinor Hoxha <rinor.hoxha@iohk.io>, Mikhail Zabaluev <mikhail.zabaluev@iohk.io>
+Comments-URI: https://forum.cardano.org/t/cip-catalyst-registration-metadata-format/44038
+Status: Draft
+Type: Standards
+Created: 2020-01-05
+License: CC-BY-4.0
+---
+
+## Abstract
+
+Cardano uses a sidechain for its treasury system. One needs to "register" to participate on this sidechain by submitting a registration transaction on the mainnet chain. This CIP details the registration transaction format.
+
+## Motivation
+
+Cardano uses a sidechain for its treasury system ("Catalyst"). One of the desirable properties of this sidechain is that even if its safety is compromised, it doesn't cause loss of funds on the main Cardano chain. To achieve this, instead of using your wallet's recovery phrase on the sidechain, we need to use a brand new "voting key".
```

Comment:
Conflicted: the treasury is "on the main chain" and does not directly  AFAIK go through the Catalyst, which I feel is a voting mechanism for treasury disbursement - but the funding itself is from/to Cardano via MIR tx isn't it?